baiji/serialization/yaml.py

```diff
@@ -14,20 +16,37 @@ def load(f, *args, **kwargs):
 
 
 def loads(s, *args, **kwargs):
-    import yaml
-    return yaml.load(s, *args, **kwargs)
+    return _load(s, *args, **kwargs)
 
 
 def dumps(obj, *args, **kwargs):
     import yaml
+    safe_dump = kwargs.pop('safe_dump', False) # Perhaps default should be True?
+    if safe_dump:
```

Comment:
As an alternative, I propose the following:
```py
safe_mode = kwargs.pop('safe', None) # This becomes True in the future
try:
    return yaml.safe_dump(obj, *args, **kwargs)
except: yaml.representer.RepresenterError as e:
    if safe_mode:
        raise SerializationSafetyError(*e.args)
    if safe_mode is None:
        warnings.warn("Unsafe YAML serialization. This will generate an error in the future. Call with `safe=False` if you really want unsafe serialization.", warnings.DeprecationWarning, stacklevel=2)
    # if safe_mode is None (not given) or False (explicitly set), fall back to unsafe behavior
    return yaml.dump(obj, *args, **kwargs)
```